src/nls/de/menu.json

```diff
@@ -30,7 +30,24 @@
         },
         "OPEN": "Öffnen…",
         "OPEN_RECENT": {
-            "$MENU": "Letzte Dateien öffnen"
+            "$MENU": "Letzte Dateien öffnen",
+            "16": "16",
```

Comment:
I think these numbers crept into our dictionaries incorrectly. We'll probably remove them later, but they're harmless for now.